tests/e2e/upgrade/upgrade.wdio-spec.js

```diff
@@ -70,6 +71,12 @@ describe('Performing an upgrade', () => {
     });
   });
 
+  // TODO Enable this test after 4.6.0 is released
```

Comment:
If not yet, please create a ticket in milestone 4.7

Reply:
I was planning to create the ticket after this gets merged